subworkflows/local/read_alignment_rna/main.nf

```diff
@@ -0,0 +1,217 @@
+//
+// Align RNA reads
+//
+
+import Constants
+import Utils
+
+include { GATK4_MARKDUPLICATES } from '../../../modules/nf-core/gatk4/markduplicates/main'
+include { SAMBAMBA_MERGE       } from '../../../modules/local/sambamba/merge/main'
+include { SAMTOOLS_SORT        } from '../../../modules/nf-core/samtools/sort/main'
+include { STAR_ALIGN           } from '../../../modules/local/star/align/main'
+
+workflow READ_ALIGNMENT_RNA {
+    take:
+    // Sample data
+    ch_inputs         // channel: [mandatory] [ meta ]
+
+    // Reference data
+    genome_star_index // channel: [mandatory] /path/to/genome_star_index/
+
+    main:
+    // Channel for version.yml files
+    // channel: [ versions.yml ]
+    ch_versions = Channel.empty()
+
+    // Sort inputs
+    // channel: [ meta ]
+    ch_inputs_sorted = ch_inputs
+        .branch { meta ->
+            def has_existing = Utils.hasExistingInput(meta, Constants.INPUT.BAM_RNA_TUMOR)
+            runnable: Utils.hasTumorRnaFastq(meta) && !has_existing
+            skip: true
+        }
+
+    // Create FASTQ input channel
+    // channel: [ meta_fastq, fastq_fwd, fastq_rev ]
+    ch_fastq_inputs = ch_inputs_sorted.runnable
+        .flatMap { meta ->
+            def meta_sample = Utils.getTumorRnaSample(meta)
+            meta_sample
+                .getAt(Constants.FileType.FASTQ)
+                .collect { key, fps ->
+                    def (library_id, lane) = key
+
+                    def meta_fastq = [
+                        key: meta.group_id,
+                        id: "${meta.group_id}_${meta_sample.sample_id}",
+                        sample_id: meta_sample.sample_id,
+                        library_id: library_id,
+                        lane: lane,
+                    ]
+
+                    return [meta_fastq, fps['fwd'], fps['rev']]
+                }
+        }
+
+    //
+    // MODULE: STAR alignment
+    //
+    // Create process input channel
+    // channel: [ meta_star, fastq_fwd, fastq_rev ]
+    ch_star_inputs = ch_fastq_inputs
+        .map { meta_fastq, fastq_fwd, fastq_rev ->
+            def meta_star = [
+                *:meta_fastq,
+
+
+                // TODO(SW): understand target format
```

Comment:
A cryptic TODO here.

Reply:
- Removed in https://github.com/nf-core/oncoanalyser/pull/67